docs/.wordlist.txt

```diff
@@ -47,3 +47,22 @@ YAML
 STSStack
 sshuttle
 vpn
+behavior
+HH
+YYYY
+ss
+SQLAlchemy
+Caracal
+Xena
+fbcab
+ccb
+aac
+de
+acf
+ef
+ec
+PciDevice
+dev
+pci
+uuids
+opendev
```

Comment:
These changes should be moved to the .custom_wordlist.txt :

> To add exceptions for words flagged by the spelling check, edit the ``.custom_wordlist.txt`` file.
> You shouldn't edit ``.wordlist.txt``, because this file is maintained and updated centrally and contains words that apply across all projects.

Although I'm not sure how we're updating the .wordlist.txt from the central location in the projects though?